wazo_agentd/service/handler/on_queue.py

```diff
@@ -51,6 +53,12 @@ def handle_on_agent_paused(self, msg):
             'Executing on agent paused command (MemberName %s)', msg['MemberName']
         )
         pause_info = self._get_pause_info(msg)
+        if not pause_info:
+            logger.info(
```

Comment:
Avoid spamming admin with normal behavior
```suggestion
            logger.debug(
```